### DISCUSSION TITLE
Latest Release Cut Schedule

### DISCUSSION THREAD

**Discussion by ol3d** [lae/ansible-role-proxmox#257] [General]
Is there a plan for a new release to be cut soon? There are a lot of new, helpful features currently merged into dev and I am wondering what the schedule/timeline is looking like for getting those out in an official release.

**Comment by lae** [comment]
kind of open-ended but this was my last statement about cutting a release https://github.com/lae/ansible-role-proxmox/issues/243#issuecomment-2219306890

i just haven't gotten around to it yet

**Comment by lae** [comment]
I forgot I actually did start a release branch for v1.9.0 back in July when I left that comment, which is easy to miss. I'll just go ahead and open a PR for that branch for the time being until I get around to taking care of that feature request (and the metric server stuff while we're at it).

https://github.com/lae/ansible-role-proxmox/pull/263
